packages/date-picker/test/basic.test.js

```diff
@@ -71,60 +73,70 @@ describe('basic features', () => {
 
   it('should open on input tap', async () => {
     tap(input);
-    await oneEvent(datePicker.$.overlay, 'vaadin-overlay-open');
+    await oneEvent(overlay, 'vaadin-overlay-open');
```

Comment:
I'm not sure if this test still makes a lot of sense, but let's keep it for now.